x/auth/legacy/legacytx/msg_service.go

```diff
@@ -0,0 +1,32 @@
+package legacytx
+
+import (
+	gocontext "context"
+	"fmt"
+
+	"google.golang.org/grpc"
+
+	sdk "github.com/cosmos/cosmos-sdk/types"
+	sdkerrors "github.com/cosmos/cosmos-sdk/types/errors"
+)
+
+// Invoke implements the grpc ClientConn.Invoke method.
+// TODO Full amino support still needs to be added as part of https://github.com/cosmos/cosmos-sdk/issues/7541.
```

Comment:
This function doesn't invokes anything it only schedules an invoke. But I see why this is required. `NewMsgClient`  requires this interface. Let's update a comment here and also in wrappers, because it is a bit misleading, I was asking myself: _why TxBuilder is invoking something_?

Reply:
Added comment